### PR TITLE
Add .nmd to .xlsx export

### PR DESCRIPTION
**Pull request by Xarthisius**
This functionality should match whatever KLA software is natively doing. 

TODO:
- [ ] tests

### CODE REVIEW

File: src/htmdec_formats/utils/convert.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+            correction += 1
+            continue
+        worksheet.write(1, col, value["unit"])
+        data = dataset.result_vars[key].accumulator / unit
+        for j in range(ntests):
+            worksheet.write(j + 2, col, data[j])
+
+
+def to_xlsx(dataset, fname):
+    with xlsxwriter.Workbook(fname) as workbook:
```
Comment (matthewturk) [inline src/htmdec_formats/utils/convert.py]: Can you make this a method on the `IndenterDataset` object instead?

**Review by matthewturk — COMMENTED** [review]
Only two things -- can you put the `to_xlsx` function into a method on the dataset, and can you prefix the `results_cols` etc with a `_` so that they are clearly private-ish?

**Review by matthewturk — APPROVED** [review]
lgtm; lemme know what else you want to do before WIP goes away

**Comment by Xarthisius** [comment]
I wanted to add tests, but I need to figure out how to get some public NMD...

**Pull request merged**